Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,46 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    require "pty"
+
+    assert_equal "LGOGDownloader #{version}",
+      pipe_output("#{bin}/lgogdownloader --version").chomp
+
+    login_cmdline =
+      [bin/"lgogdownloader", "--list", "--retries", "1"]
+    test_credentials = {
+      :username => "test@example.com\n",
+      :password => "secret\n",
+    }
+    ENV["XDG_CONFIG_HOME"] = mktemp(:retain => true) { pwd }
```

Comment:
this seems weird. we're already in a temp directory and HOME is already set

Reply:
Can we guarantee we’re in a temp directory? I wouldn’t want test to spit files left and right unexpectedly.

Reply:
You're in `testpath` which is always a temp directory.

Reply:
Thanks for the pointer!

Reply:
… and fixed. ☺️